client/data/home/use-skip-current-view-mutation.ts

```diff
@@ -28,16 +28,21 @@ function useSkipCurrentViewMutation< TData, TError >( siteId: number ): Result<
 				{ staleTime: Infinity }
 			);
 
+			const view_name = ( data as any ).view_name;
+			const multipleCardViews = [ 'VIEW_POST_PUBLISH', 'VIEW_SITE_SETUP' ];
```

Comment:
Pretty sure this is a typo

```suggestion
			const multipleCardViews = [ 'VIEW_POST_LAUNCH', 'VIEW_SITE_SETUP' ];
```

Reply:
oops good catch!